docker/lite/Dockerfile.ubi7

```diff
@@ -1,11 +1,11 @@
 # Copyright 2019 The Vitess Authors.
```

Comment:
All the renames in this folder are weird. Did you actually use `git mv` to rename these files? Or is GH inferring them as renames because of the similarity of content?

Reply:
The latter